src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.cs

```diff
@@ -247,16 +246,14 @@ internal async Task<bool> ApplyChangedSettingsAsync(CancellationToken cancellati
             }
 
             // get package sources as ordered list
-            var packageSources = PackageSourcesListBox.Items.Cast<PackageSource>().ToList();
-            packageSources.AddRange(MachineWidePackageSourcesListBox.Items.Cast<PackageSource>().ToList());
+            List<PackageSourceContextInfo> packageSources = PackageSourcesListBox.Items.Cast<PackageSourceContextInfo>().ToList();
+            packageSources.AddRange(MachineWidePackageSourcesListBox.Items.Cast<PackageSourceContextInfo>().ToList());
```

Comment:
It is unnecessary but since this is a linq extension method I didn't want to have it use deferred execution and incur penalties each time it is accessed.